.github/workflows/lint.yml

```diff
@@ -0,0 +1,20 @@
+name: Lint
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
```

Comment:
I don't think it will work, since it won't run in others PRs that aren't made in `main` branch.
I guess it's good if you only put `on: [push, pull_request]`. What do you think?

Reply:
I think we should leave it this way. Right now this action will run on any PRs wanting to merge into main, which, in my understanding, is the production branch.
Is there really a point of running it in other branches, if they will, eventually, merge into main? 
Also, the fewer times the action is run, the fewer hours it will consume at the end of the month.

With that said, if you still want me to change it, I'll do it :)

Reply:
That is nice argument, leave it as it is and ignore my review. 😄